code/game/objects/items/twohanded.dm

```diff
@@ -346,7 +346,8 @@
 		icon_state = "dualsaber[item_color][wielded]"
 	else
 		icon_state = "dualsaber0"
-	SEND_SIGNAL(src, COMSIG_COMPONENT_CLEAN_ACT, CLEAN_STRENGTH_BLOOD)
```

Comment:
The clean_act signal needs to be readded, blood isn't the only thing that uses it